library/Versions.re

```diff
@@ -145,3 +146,20 @@ let getRemoteVersions = () => {
      )
   |> Lwt.return;
 };
+
+let throwIfInstalled = versionName => {
+  switch(getInstalledVersions()) {
+  | Ok(x) => {
+    Array.to_list(x)
+    |> List.exists(x => {
+      open Local;
+      x.name == versionName;
+    })
+    |> x => switch x {
```

Comment:
There's a command: `esy fmt` that runs `refmt` on the codebase. Once you'll run it, you see that `switch` over boolean will be converted to a ternary. It'll be prettier 😄 please run it against your code before pushing (I'm adding it as a commit hook on #30)

Reply:
I got it!